### ISSUE TITLE
MSI Installer Created Shortcut Run As Admin Issue

### ISSUE THREAD

**Issue by Wolfereign** [WhiteMagic/JoystickGremlin#82]
Due to the integration with HiDGuardian I noticed that I know need to run JoystickGremlin as Administrator so that it can make the necessary registry changes.  This is when I noticed that the shortcut that the installer creates in the start menu / desktop cannot be right clicked and run as administrator.  Neither can you check the option in it's properties to force run as administrator (it's greyed out).

When Looking at the shortcut I noticed that it does not point to the exe for joystick gremlin directly which may be the problem.

![image](https://user-images.githubusercontent.com/5723368/35928196-cdaddd72-0be9-11e8-8310-45bf69be55d7.png)

In contrast here is a shortcut created manually by right clicking the exe and selecting create shortcut. 
![image](https://user-images.githubusercontent.com/5723368/35928346-318a2684-0bea-11e8-9523-352a58aa8186.png)

I tested a few different versions of windows 10 and some can right click the start menu shortcut and run as admin while the latest version of windows cannot.  However they all had the same issue with the Run As Administrator properties option.

**Comment by WhiteMagic** [comment]
Yeah I think that's some sort of issue with how I'm telling the installer to create that shortcut. I'll have to look into what odd options Windows has when you create those.

The admin bit is only needed if you want to use hidguardian and while it's a bit inconvenient I found this the simplest way in the short term to interface with it. Though as there are quite a few changes happening with hidguardian and it's assorted helpers I'm expecting this to change in the future. So when I redo it it won't require admin rights anymore.

**Comment by izolini** [comment]
It would be nice also if the JGremlin didn't create a new icon on my desktop everytime I start it. (I've already moved the icon where I want it, but JGremlin always create a new one on my desktop). Annoying.

**Comment by WhiteMagic** [comment]
Was it the installer creating a new icon or launching Gremlin itself that did that. If it's the installer then there might not be a way for me to check for the existence of that but you shouldn't need to launch the installer all the time. If it's the Gremlin program itself then I'm confused as I've never made it create shortcuts of any kind.

**Comment by izolini** [comment]
The icon created by installer was doing this. After install, I moved the desktop icon (created by installer) to start menu. Everytime I started the program using that icon, it created again the desktop icon. I wasn't allowed to edit any of them (couldn't alter the program pointer), so I deleted the desktop and start icons and created a new one "by hand", pointing to the .exe, and now I don't have this problem anymore. 
Cheers.
